docs/content/latest/api/ysql/the-sql-language/statements/cmd_analyze.md

```diff
@@ -0,0 +1,139 @@
+---
+title: ANALYZE statement [YSQL]
+headerTitle: ANALYZE
+linkTitle: ANALYZE
+description: Collect statistics about database tables with the ANALYZE statement.
+beta: /latest/faq/general/#what-is-the-definition-of-the-beta-feature-tag
+menu:
+  latest:
+    identifier: cmd_analyze
+    parent: statements
+aliases:
+  - /latest/api/ysql/commands/cmd_analyze/
+isTocNested: true
+showAsideToc: true
+---
+
+## Synopsis
+
+ANALYZE collects statistics about the contents of tables in the database, and stores the results in the `pg_statistic` system catalog. Subsequently, the query planner uses these statistics to help determine the most efficient execution plans for queries.
+
+The Yugabyte implementation is based on the framework provided by PostgreSQL, which requires the storage layer to provide random sample of rows of predefined size. The size is calculated based on number of factors, such as data types of included columns.
+
+{{< note title="Note" >}}
+Currently sampling algorithm is not optimized fror a large table. It may take minutes to collect statistics from a table containing several million rows of data.
```

Comment:
```suggestion
The sampling algorithm is not currently optimized for large tables. It may take several minutes to collect statistics from a table containing millions of rows of data.
```